SPECS/dracut/fix-grep-selinux-execmem.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/dracut-functions.sh b/dracut-functions.sh
```

Comment:
Can this be a git patch format that also has comments for why this patch is needed?

Reply:
Just submitted the PR upstream, will use that commit instead.